examples/services/package.json

```diff
@@ -11,43 +11,50 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test"
   },
   "devDependencies": {
-    "@ember/jquery": "^1.1.0",
     "@ember/optional-features": "^2.0.0",
+    "@glimmer/component": "^1.0.2",
+    "@glimmer/tracking": "^1.0.2",
+    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-ajax": "^5.0.0",
+    "ember-auto-import": "^1.6.0",
     "ember-cli": "~3.22.0",
     "ember-cli-app-version": "^4.0.0",
-    "ember-cli-babel": "^7.23.0",
-    "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-babel": "^7.22.1",
+    "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-htmlbars": "^5.3.1",
     "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^2.0.2",
-    "ember-cli-uglify": "^3.0.0",
-    "ember-data": "~3.10.0",
+    "ember-cli-terser": "^4.0.0",
+    "ember-data": "~3.22.0",
     "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.2",
+    "ember-fetch": "^8.0.2",
+    "ember-load-initializers": "^2.1.1",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^4.6.0",
     "ember-resolver": "^8.0.2",
-    "ember-source": "~3.23.1",
-    "ember-welcome-page": "^4.0.0",
-    "eslint-plugin-ember": "^7.7.2",
+    "ember-source": "~3.22.0",
+    "ember-template-lint": "^2.14.0",
+    "ember-test-selectors": "^4.1.0",
```

Comment:
I'd rather not introduce this here. The idea is to keep the examples as simple as possible.